src/components/feedback/__tests__/__snapshots__/Loader.test.js.snap

```diff
@@ -0,0 +1,13 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<Loader /> should not render the loader when show is false 1`] = `
+<ContextConsumer>
+  <Component />
+</ContextConsumer>
+`;
+
+exports[`<Loader /> should render loader when show is true 1`] = `
+<ContextConsumer>
+  <Component />
+</ContextConsumer>
+`;
```

Comment:
I've had this happen a few times too. For snapshots you need to make sure to import the un-connected component. Otherwise the output is a bit meaningless